packages/fxa-payments-server/src/components/PlanDetails/index.tsx

```diff
@@ -0,0 +1,91 @@
+import React, { useState } from 'react';
+import { Localized } from 'fluent-react';
+import { formatCurrencyInCents } from '../../lib/formats';
+
+import './index.scss';
+import { Plan } from '../../store/types';
+
+type PlanDetailsProps = {
+  selectedPlan: Plan;
+  showExpandButton?: boolean;
+  className?: string;
+};
+
+export const PlanDetails = ({
+  selectedPlan,
+  showExpandButton = false,
+  className = 'default',
+}: PlanDetailsProps) => {
+  const initialDetailsHiddenState = showExpandButton ? true : false;
+  const [detailsHidden, setDetailsState] = useState(initialDetailsHiddenState);
+  const { product_name, amount, interval } = selectedPlan;
+
+  return (
+    <section className={`container card plan-details-component ${className}`}>
+      <div className="plan-details-header">
+        <div className="plan-details-header-wrap">
+          <div className="plan-details-logo-wrap">
+            <img src="https://accounts-static.cdn.mozilla.net/product-icons/fpn.svg" alt={product_name} data-testid="product-logo" />
```

Comment:
I pinged Les over Slack about this since hardcoding the URL didn't feel quite right, he said it should come from Stripe metadata and linked me here:

https://github.com/mozilla/fxa/blob/master/packages/fxa-payments-server/src/routes/Product/SubscriptionUpgrade/index.tsx#L231

PR looks great to me otherwise, thanks for all the work/rework!